openapi/components/parameters/imageResize.yaml

```
@@ -0,0 +1,8 @@
+name: imageResize
```

I believe `resize` is not quite correct word to use in param name. 

```suggestion
name: imageSize
```

---

We resize an original image